nss-day-cohort-41/react-nutshell-thepecans
# changed button to checkbox

# Description
changed "mark as complete/incomplete" button to checkbox in tasks feature.  All other functionality is the same.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
check the box in tasks and verify it has been moved to completed tasks page.  Uncheck the box to verify the inverse.
# Checklist:
